components/Breadcrumb.tsx

```diff
@@ -0,0 +1,91 @@
+import { FC, useEffect, useState } from 'react'
+import { Breadcrumbs, NavList } from '@primer/react'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import { INavbarElement } from 'lib/interfaces'
+import Image from 'next/image'
+
+const Breadcrumb: FC = () => {
+    const router = useRouter()
+    const [open, setOpen] = useState(false)
+    const [breadcrumbs, setBreadcrumbs] = useState<INavbarElement[]>([])
+    const isCM = (path: string): boolean => {
+        return router.query.candy_machine_ID === path
+    }
+
+    useEffect(() => {
+        if (router) {
+            const linkPath = router.asPath.split('/')
+            linkPath.shift()
+            const pathArray = linkPath?.map((path, i) => ({
+                title: isCM(path) ? '[CM]: ' + path.substring(0, 12).concat('...') : path,
+                url: '/' + linkPath.slice(0, i + 1).join('/'),
+            }))
+            pathArray.unshift({ title: 'Dashboard', url: '/' })
+            setBreadcrumbs(pathArray)
+        }
+    }, [router])
+
+    return (
+        <>
+            <Breadcrumbs className='d-none d-md-flex flex-row ml-3'>
+                {breadcrumbs.map((element) => {
+                    return (
+                        <Link href={element.url} key={element.url}>
+                            <Breadcrumbs.Item
+                                href={element.url}
+                                key={element.url}
```

Comment:
I think we should remove also the key and just use it in the Link. Also use element.title instead of element.url since I was getting 2 equal element.url 
![image](https://user-images.githubusercontent.com/43625217/181281794-8daa3caa-42dd-40fc-8ed8-e72fd1b8fedf.png)